pygmt/src/subplot.py

```diff
@@ -0,0 +1,203 @@
+        **b**ottom. Considerations for **C**: Use when all subplots in a
+        **C**\ olumn share a common *x*-range. The first (i.e., **t**\ op) and
+        the last (i.e., **b**\ ottom) rows will have *x* annotations; append
+        **t** or **b** to select only one of those two rows [both]. Append
+        **+l** if annotated *x*-axes should have a label [none]; optionally
+        append the label if it is the same for the entire subplot. Append
+        **+t** to make space for subplot titles for each row; use **+tc** for
+        top row titles only [no subplot titles]. Labels and titles that depends
+        on which row or column are specified as usual via a subplot's own
+        **frame** setting. Considerations for **R**: Use when all subplots in a
+        **R**\ ow share a common *y*-range. The first (i.e., **l**\ eft) and
+        the last (i.e., **r**\ ight) columns will have *y*-annotations; append
+        **l** or **r** to select only one of those two columns [both]. Append
+        **+l** if annotated *y*-axes will have a label [none]; optionally,
+        append the label if it is the same for the entire subplot. Append
+        **+p** to make all annotations axis-parallel [horizontal]; if not used
+        you may have to set **clearance** to secure extra space for long
+        horizontal annotations. Append **+w** to draw horizontal and vertical
+        lines between interior panels using selected pen [no lines].
+    title : str
+        While individual subplots can have titles (see **layout** or
+        **frame**), the entire figure may also have an overarching *heading*
+        [no heading]. Font is determined by setting :gmt-term:`FONT_HEADING`.
+    {V}
+    {XY}
+    """
+    kwargs = self._preprocess(**kwargs)  # pylint: disable=protected-access
+
+    with Session() as lib:
+        try:
+            arg_str = " ".join(["begin", f"{nrows}x{ncols}", build_arg_string(kwargs)])
+            lib.call_module(module="subplot", args=arg_str)
+            yield
+        finally:
+            v_arg = build_arg_string({"V": kwargs.get("V")})
+            lib.call_module("subplot", f"end {v_arg}".strip())
+
+
+@fmt_docstring
+@contextlib.contextmanager
+@use_alias(A="fixedlabel", C="clearance", V="verbose")
+def sca(self, ax=None, **kwargs):
+    r"""
+    Set the current Axes instance to *ax*.
+
+    Before you start plotting you must first select the active subplot. Note:
+    If any *projection* option is passed with **?** as scale or width when
+    plotting subplots, then the dimensions of the map are automatically
+    determined by the subplot size and your region. For Cartesian plots: If you
+    want the scale to apply equally to both dimensions then you must specify
+    ``projection="x"`` [The default ``projection="X"`` will fill the subplot by
+    using unequal scales].
+
+    {aliases}
+
+    Parameters
+    ----------
+    ax : str
+        *row,col*\|\ *index*.
```

Comment:
I think it might be good to allow list/tuple inputs like `fig.sca(ax=[0, 1], ...)`, but `@kwargs_to_string(ax="sequence_comma")` won't work here because of #640.

Reply:
Ok, `fig.sca(ax=[0, 1], ...)` works now in commit 6eb58eada1e7ff6ac2aa0405470ee370523f81b1. Did a one-liner workaround using `ax = ",".join(map(str, ax)) if is_nonstr_iter(ax) else ax`.